src/pages/StockListing/StockListing.tsx

```diff
@@ -16,14 +16,16 @@ export type Stock = {
   type: string;
 };
 
+const finnhubURL = 'https://finnhub.io/api/v1'
+
 //Todo: Add proper pagination
 const StockListing = (): JSX.Element => {
   const [data, setData] = useState<Stock[] | []>([]);
   const howManyToDisplay: number = 10;
 
   useEffect(() => {
     const fetchData = async () => {
-      const rawData = await fetch(process.env.REACT_APP_STOCK_URL!);
+      const rawData = await fetch(`${finnhubURL}/stock/symbol?exchange=US${process.env.REACT_APP_STOCK_TOKEN}`!);
```

Comment:
We could even eliminate the separate token variable in `.env` and just have something like this:
```const rawData = await fetch(`${finnhubURL}/stock/symbol?exchange=US&token=${process.env.REACT_APP_STOCK_KEY}`!);```

Reply:
Thanks for the suggestion!! I've gone and removed the token env var and added the token query to the finnhub API call